content/blog/check_my_repo.md

```diff
@@ -0,0 +1,150 @@
+
+#### Readme
+
+A friendly README and clear code examples will make it easier for anyone who lands on your project to get started. It is about knowing your audience and their needs. The [Open Source Guide](https://opensource.guide/starting-a-project/#writing-a-readme) reminds us that the README is more than just a set of instructions, it&#39;s a place to talk about the project goals, product vision, and roadmap, which serves as a compass to have a constructive discussion.
+
+#### Changelog
+
+The main purpose of this rule is to make it easier for users and contributors to see precisely what notable changes have been made between each release (or version) of the project. While Changelog is a file, a SemVer, a semantic version, may be a better choice in systems with many dependencies. Doesn&#39;t matter what you choose, this makes it easier for people to understand the development speed of the project, or it&#39;s lack of maintenance. Check-my-repo searches if the project has a CHANGELOG.md file in the project root or if it contains a release tag on GitHub.
+
+#### Contributing
+
+A guideline saves lots of time from both maintainers and contributors. Eager contributors rely on this file to be their guiding hand. The more descriptive, with understandable and commented examples, perhaps using tutorials and other strategies considering different cognitive aspects of the possible contributor (while some people prefer text, some video, some audio...), the easier and faster it will be for your contributor to actively incorporate themselves into the project.
+
+#### Ownership
+
+The main purpose of this file is to make sure that someone is put in charge of the project and to prevent it from being forgotten. Another reason is to provide a contact for potential contributors. When no one other than the original author has maintenance rights, continuation of the project through bifurcation can lead to confusion about the status and reliability of the project.
+
+#### Test Directory
+
+Having tests on your project can avoid regresion, breaks and many other issues. A few reasons why testing is important: when bugs are caught in the earlier stages it costs much less to fix them; you add security; helps assure quality and digital confidence; which leads to users satisfaction. If your project has an automated test set, it will help to have changes incorporated into your project more quickly, easing the burden on reviewers and maintainers.
+
+#### Code of Conduct
+
+A code of conduct establishes expectations and facilitates a healthy and constructive community.
+
+_When a project seems hostile or unwelcoming, even if it&#39;s just one person whose behavior is tolerated by others, you risk losing many more contributors, some of whom you may never even meet. It&#39;s not always easy to adopt or enforce a code of conduct, but fostering a welcoming environment will help your community grow._
```

Comment:
instead of "some of whom you may never even meet" maybe be more straightforward like _you risk alienating potential future contributors_